app-frontend/yarn.lock

```diff
@@ -4108,11 +4108,24 @@ lcid@^1.0.0:
   dependencies:
     invert-kv "^1.0.0"
 
-leaflet-draw@~0.4.9:
+leaflet-draw-drag@^0.4.4:
```

Comment:
After running yarn, this file was changed. Re run yarn to update the lockfile and commit it